### PR TITLE
#1-4Task. Alexey.Fedko

### PR DESCRIPTION
**Pull request by AlexeiFedya**

### CODE REVIEW

**Comment by AlexeiFedya** [comment]
https://travis-ci.org/github/AlexeiFedya/js-assignments/builds/703872458

File: task/01-strings-tasks.js [hunk 1]
```diff
@@ -238,7 +251,12 @@ function encodeToRot13(str) {
  *   isString(new String('test')) => true
  */
 function isString(value) {
-    throw new Error('Not implemented');
+    if (typeof value === 'string'  || value instanceof String){
+        return true
+        }
+    else{
```
Comment (ilya4321) [inline task/01-strings-tasks.js:257]: if/else is redundant here

File: test/03-date-tests.js [hunk 2]
```diff
@@ -83,10 +83,6 @@ describe('03-date-tasks', function() {
                 startDate: new Date(2000,1,1,10,0,0),
                 endDate:   new Date(2000,1,1,15,20,10,453),
                 expected:  '05:20:10.453'
-            }, {
-                startDate: new Date(2000,1,1,10,0,0),
-                endDate:   new Date(2000,1,2,15,20,10,453),
-                expected:  '29:20:10.453'
             }
```
Comment (ilya4321) [inline test/03-date-tests.js]: WOW! Please revert it. All tests are correct 
https://www.youtube.com/watch?v=rR4n-0KYeKQ

**Review by IlyaMokin — COMMENTED** [review]
looks good in general, but please fix the last issue with broken test

File: test/03-date-tests.js (hunk 2 above)
Comment (AlexeiFedya) [reply test/03-date-tests.js]: > WOW! Please revert it. All tests are correct
> https://www.youtube.com/watch?v=rR4n-0KYeKQ

hello Ilya, I revert test3 , and edit task3, npm test is positive, but travic-ci is failed, why?

Comment (AlexeiFedya) [reply test/03-date-tests.js]: i creat pr task5-9 and similar failed from travic-ci

Comment (AlexeiFedya) [reply test/03-date-tests.js]: Please provide me answers to my questions. i look to hearing from you soon

**Comment by AlexeiFedya** [comment]
i edit tesk 3 and task 3, npm test is positive, but travic ci is failed, why ?

**Comment by IlyaMokin** [comment]
can be failed on our travis account because of additional eslint test, on your account should be fine.

**Pull request closed without merging**